code/modules/clothing/suits/toggles.dm

```diff
@@ -51,6 +51,7 @@
 	for(var/X in actions)
 		var/datum/action/A = X
 		A.UpdateButtonIcon()
+	return hood
```

Comment:
This will always be null no matter what, even if you didn't have a hood in the first place.